annlite/executor.py

```diff
@@ -273,6 +277,30 @@ def search(
 
             flat_docs.match(self._index, **match_args)
 
+    @requests(on='/backup')
+    def backup(self, target: Optional[str] = None, **kwargs):
```

Comment:
```suggestion
    def backup(self, parameters: Dict, **kwargs):
        target = parameter.get('target')
        ...
```

The executor endpoint function only works with `parameters`